src/MongoDB.Driver.Core/Core/Servers/ServerMonitor.cs

```diff
@@ -93,7 +93,7 @@ public void Initialize()
 
         public void Invalidate(string reasonInvalidated)
         {
-            SetDescription(_baseDescription.With($"InvalidatedBecause:{reasonInvalidated}", lastUpdateTimestamp: DateTime.UtcNow));
+            SetDescription(_baseDescription.With($"InvalidatedBecause:{reasonInvalidated}", lastUpdateTimestamp: DateTime.UtcNow, type: ServerType.Unknown));
```

Comment:
As far as I can see this change is not required, because the _baseDescription.Type is already `Unknown`.

Reply:
Ah true, we're updating the _baseDescription which already has "Unknown". I think it's clearer to have the "Unknown" in the `With` statement (as it does no harm, but am happy to roll this back if you feel strongly about it :)

Reply:
I see your point, but I would remove this change since the idea of `_baseDescription` is always have initial values. Setting the field which is already has been assigned confuses me a bit. If we want to highlight `Unknown` type value, maybe we can just add a comment about _baseDescription values and how this value is expected to be used? I will rely on your and @rstam opinion here

Reply:
I'm fine with reverting this: reverted!